python/tskit/tables.py

```diff
@@ -2795,3 +2797,18 @@ def union(
             self.provenances.add_row(
                 record=json.dumps(provenance.get_provenance_dict(parameters))
             )
+
+    def find_ibd(self, samples, max_time=None, min_length=0):
+        """
```

Comment:
Better to have no docstring. If this is included, then this function will be included on read the docs (which we don't want it to be, yet.